.github/actions/setup-project/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Setup Novu Monorepo
+
+description: Sets up the whole monorepo and install dependencies
+
+inputs:
+  cypress_version:
+    description: 'The version of cypress to install'
+    required: false
+    default: ''
+outputs:
+  cypress_cache_hit:
+    description: 'Did cypress used binary cache'
+    value: ${{ inputs.cypress_version != '' && steps.cache-cypress-binary-version.outputs.cache-hit || steps.cache-cypress-binary.outputs.cache-hit}}
```

Comment:
Where I could read about the meaning of this steps?
cache-cypress-binary-version
cache-cypress-binary

Reply:
This is mainly for the old widget package that uses and old version for cypress that we need to cache binaries pinned to a particular version. Unfortuently couldn't find a nice way to handle this